.gitignore

```diff
@@ -2,5 +2,8 @@
 /TODO
 .DS_Store
 external/jazz/node_modules
+external/micro/*
```

Comment:
This might lead to confusion later on. Try making a .gitignore in the micro directory.